packages/moonstone/IconButton/IconButton.js

```diff
@@ -128,7 +128,7 @@ const IconButtonBase = kind({
  */
 const IconButtonDecorator = compose(
 	Pure,
-	TooltipDecorator({tooltipDestinationProp: 'tooltipNode'}),
+	TooltipDecorator({tooltipDestinationProp: 'decoration'}),
```

Comment:
Do we have other references to `tooltipNode` that need to be cleaned up with this change?

Reply:
It's possible this should just stay `tooltipNode`. I was thinking that a more generic term would be better, but maybe that makes things more confusing? Open to suggestions/input.

Reply:
I think it should use whatever your proposed prop is but `tooltipNode` is referenced elsewhere in this file so that should be addressed.

Reply:
I am waffling back and forth on whether this should keep its name or de updated to a generic name. I think `decoration` may be too open-ended and we may run into future issues when we try to insert some other node into this one; it's just my current preference. I am (was) hoping to get some other opinions in here to help me decide the correct course of action. In any case, the other mentions of this term should certainly be updated. Thanks for pointing that out. I didn't notice or realize that there were other mentions of it.